gemrb/core/Image.cpp

```diff
@@ -36,15 +36,15 @@ Image::~Image()
 Sprite2D* Image::GetSprite2D()
 {
 	union {
-		Color color;
+		unsigned char color[sizeof(ieDword)];
```

Comment:
Codacy found an issue: [union member 'Anonymous0::color' is never used.](https://app.codacy.com/manual/gemrb/gemrb/pullRequest?prid=6310305)